projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1688,11 +1688,11 @@ public static function wrap_message_in_html_tags( $title, $response_link, $form_
 		 *
 		 * @module contact-form
 		 *
-		 * @since 0.18.0
+		 * @since 12.2
```

Comment:
Since this is in a package, it uses the package version and not the plugin version. `0.18.0` is okay here.

```suggestion
		 * @since 0.18.0
```

Reply:
oh, thanks